static/opensearch.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/">
```

Comment:
Looking at https://docs.mongodb.com/osd.xml can we add the following fields:
- Image
- Long Name
- Developer
- Contact
- <Url rel="self" ... />